docs/releasing/publishing.md

```diff
@@ -1,17 +1,33 @@
-# Publishing GOV.UK Frontend
+# Steps you must complete before you release a new version of GOV.UK Frontend
 
-1. Make sure you have cards in the "Backlog" column to:
-  - Update the GOV.UK Design System to use the latest release
-  - Update the GOV.UK Prototype Kit to use the latest release
-  - Update the GOV.UK Frontend Docs to use the latest release
+1. Raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
+  - create release notes for the new release (example card: [#1986](https://github.com/alphagov/govuk-frontend/issues/1986))
+  - create release notes for the new version of GOV.UK Prototype Kit (example card: [#958](https://github.com/alphagov/govuk-prototype-kit/issues/958))
+  - create a card for the new release of GOV.UK Frontend (example card: [#1987](https://github.com/alphagov/govuk-frontend/issues/1987))
+  - update the GOV.UK Design System to use the new release of GOV.UK Frontend (example card: [#1347](https://github.com/alphagov/govuk-design-system/issues/1347))
+  - update the GOV.UK Frontend Docs to use the new release of GOV.UK Frontend (example card: [#75](https://github.com/alphagov/govuk-frontend-docs/issues/75))
+  - create a card for the new release of GOV.UK Prototype Kit (example card: [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917))
+  - update the GOV.UK Prototype Kit to use the new release (example card: [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923))
 
-2. Checkout **master** and pull latest changes.
+2. Add the issues to the [Design System sprint board](https://github.com/orgs/alphagov/projects/4).
 
-3. Run `nvm use` to ensure you are using the right version of Node.js and npm.
+3. Write a summary to post on Slack after the release. For example:
 
-4. Run `npm install` to ensure you have the latest dependencies installed.
+    🚀 We’ve just released GOV.‌UK Frontend v3.7.0. It's now easier and faster to use our Sass. We've also made improvements to back links, breadcrumbs, lists and the header. Thanks to @SLACK-NAME for helping with this release. [https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0)
```

Comment:
Something which came up in the 3.10.0 release is that we have separate release notes for the release of the prototype kit. Is that something we could mention here?

Reply:
@vanitabarrett Yeah, that sounds good. Maybe we could reword this step as follows?

"Write summaries to post on Slack after the release of both Frontend and Prototype Kit. For example:
🚀 We’ve just released GOV.‌UK Frontend v3.7.0. It's now easier and faster to use our Sass. We've also made improvements to back links, breadcrumbs, lists and the header. Thanks to @SLACK-NAME for helping with this release. https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0
Remember that Prototype Kit needs its own summary, since it has its own release notes."

Reply:
```suggestion
    🚀 We’ve just released GOV.‌UK Frontend v3.7.0. It's now easier and faster to use our Sass. We've also made improvements to back links, breadcrumbs, lists and the header. Thanks to @SLACK-NAME for helping with this release. [https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0)
Remember that Prototype Kit needs its own summary, since it has its own release notes.    
```